packages/commonwealth/client/scripts/navigation/Router.tsx

```diff
@@ -10,12 +11,38 @@ import { PageNotFound } from 'views/pages/404';
 import CommonDomainRoutes from './CommonDomainRoutes';
 import GeneralRoutes from './GeneralRoutes';
 
-const Router = (customDomain: string) =>
-  createBrowserRouter(
+export type RouteFeatureFlags = {
+  proposalTemplatesEnabled: boolean;
+  newAdminOnboardingEnabled: boolean;
+  communityHomepageEnabled: boolean;
+};
+
+const Router = (customDomain: string) => {
+  const client = OpenFeature.getClient();
```

Comment:
cant we use `useFlag` here? Router is just another component so it should be possible? Or there are any other constraints?

Reply:
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/5ab3378e-9529-473e-a9a1-ca87f8ec596e)

probably not possible

Reply:
Yeah we cannot use it here because it is not being built as a react component so it wont have context which is how the client is passed around.